qgroundcontrol.qrc

```diff
@@ -247,4 +244,8 @@
     <qresource prefix="/res/audio">
         <file alias="Alert">resources/audio/alert.wav</file>
     </qresource>
+    <qresource prefix="/AutopilotPlugins/PX4">
```

Comment:
Unit test is failing because of change in case for AutoPilotPlugins->AutopilotPlugins.